packages/webapp/src/pages/collection.tsx

```diff
@@ -1,8 +1,62 @@
 import { type NextPage } from "next";
 import Head from "next/head";
 import { Navbar } from "../components/navbar";
+import React, { useState, useEffect } from 'react';
+import { useInventoryGetCollection } from "../generated";
+import { useAccount } from 'wagmi'
+import { deployment } from "src/deployment";
+// Eventually, you have to get all the effects used in the collection's cards
+const effects = [{label: 'Charge', active: false}, {label: 'Flight', active: false}, {label: 'Courage', active: false}, {label: 'Undying', active: false}, {label: 'Frenzy', active: false}, {label: 'Enlightened', active: false}];
+const types = [{label: 'Creature', active: false}, {label: 'Magic', active: false}, {label: 'Weapon', active: false}];
```

Comment:
Everything is visible since the filtering is not activated by default. The main problem is that the cards that are minted are mainly identical: they have the same name and don't have effects and types yet. I'll try to deal with that after fixing the rest of the collection.

Reply:
Hhm I see, do we can either say: by default all the filters are on `true`, or by default, filtering is off and we when we switch it on we only want to display one thing. I've definitely seen both on websites. I guess the way you implemented it makes more sense here.